db/src/migration.rs

```diff
@@ -1,81 +0,0 @@
-use crate::{
```

Comment:
Moved to [db-migration/src/lib.rs](https://github.com/nervosnetwork/ckb/pull/2119/files#r443936479)